### ISSUE TITLE
Add an "about" command

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#299]
```
Add an "about" command (or "help:about") to show useful information about the 
installed version of gtalksms, like version, release date, etc.
```

Original issue reported on code.google.com by `jaime...@gmail.com` on 10 Oct 2012 at 9:00

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision d186e945f736.
```

Original comment by `yako...@gmail.com` on 21 Sep 2013 at 2:53
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4.4.1
```

Original comment by `Florent....@gmail.com` on 6 Oct 2013 at 7:30
- Changed state: **Deployed**
